linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/bash/shared.sh

```diff
@@ -1,5 +1,9 @@
-
 # platform = multi_platform_all
+# reboot = false
```

Comment:
Same as above, reboot should be true since this is dealing with GRUB?

Reply:
Thanks for noting changed it in 78f7c4112d5f573d0e9327af8465d0b5669696ac